### PR TITLE
Fix bug where multiselection was canceled on menu click.

### LINKED ISSUE #566: Selecting multiple cells and then clicking on the menu bar clears the selection
Title says it all.

Ping @ellisonbg, who also is testing multiselect today.

This is the underlying cause of one of the points in #558

### PR DESCRIPTION
**Pull request by jdfreder**
partially addresses #558 
closes #566

### CODE REVIEW

File: notebook/static/notebook/js/notebook.js [hunk 1]
```diff
@@ -660,26 +660,43 @@ define(function (require) {
...
-                if (this.mode !== 'command') {
-                    this.command_mode();
-                }
-            }
-            var current_selection = this.get_selected_cells();
-            for (var i=0; i<current_selection.length; i++) {
-                current_selection[i].unselect();
+    Notebook.prototype.select = function(indicies) {
+        
+        // Make sure indicies are an array
```
Comment (jasongrout) [inline notebook/static/notebook/js/notebook.js]: "indices"

File: notebook/static/notebook/js/notebook.js [hunk 2]
```diff
@@ -660,26 +660,43 @@ define(function (require) {
...
+        if (!$.isArray(indicies)) {
+            indicies = [indicies];
+        }
+        
+        // Remove invalid indicies.
+        indicies = indicies.filter(this.is_valid_cell_index, this);
```
Comment (jasongrout) [inline notebook/static/notebook/js/notebook.js]: "indices"

File: notebook/static/notebook/js/notebook.js [hunk 3]
```diff
@@ -660,26 +660,43 @@ define(function (require) {
...
-                }
-            }
-            var current_selection = this.get_selected_cells();
-            for (var i=0; i<current_selection.length; i++) {
-                current_selection[i].unselect();
+    Notebook.prototype.select = function(indicies) {
+        
+        // Make sure indicies are an array
+        if (!$.isArray(indicies)) {
+            indicies = [indicies];
```
Comment (jasongrout) [inline notebook/static/notebook/js/notebook.js]: "indices"

File: notebook/static/notebook/js/notebook.js [hunk 4]
```diff
@@ -660,26 +660,43 @@ define(function (require) {
     /**
      * Programmatically select a cell.
      * 
-     * @param {integer} index - A cell's index
+     * @param {integer||integer[]} [indicies] - A cell's index or list of indicies
```
Comment (jasongrout) [inline notebook/static/notebook/js/notebook.js]: "indices" - also misspelled various places below in code or comments.

Comment (jdfreder) [inline notebook/static/notebook/js/notebook.js]: :heavy_check_mark:

**Comment by jdfreder** [comment]
Closing in favor of #577

**Pull request closed without merging**